### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by Scoder12**
Couple things from the last PR

### CODE REVIEW

File: src/bin/luarocks [hunk 1]
```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env lua
 
---package.loaded["luarocks.core.hardcoded"] = { SYSCONFDIR = [[/nix/store/y6ck6hwqyl0a9731dmv6a3q3dxc0xl37-luarocks-3.9.0/etc/luarocks]] }
```
Comment (teto) [inline src/bin/luarocks:3]: had seen this one but didn't want to nag you too much ^^

**Review by teto — APPROVED** [review]
I am trying to give you merge permissions on this repo but can't on my own. When it's done you can merge this on your own terms

**Comment by teto** [comment]
you should have received an invitation

**Comment by Scoder12** [comment]
Got it, thanks!

**Pull request merged**